### DISCUSSION TITLE
Creating new user

### DISCUSSION THREAD

**Discussion by AunErki** [broadinstitute/seqr#3969] [Q&A]
Hi.

I recently got my new seqr docker install up and running. So far so good. Now I would like to add new users in addition to the superuser. How I can do that? Is there any instruction page for that? In older local install, there was a tab "Create User" under  "Staff Pages", but the latest docker install is missing that tab entirely.

Erki

**Comment by hanars** [comment]
users are created in seqr by adding them to a project
